cmd/rukpakctl/cmd/root.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+*/
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
```

Comment:
I think my preference would be to have a constructor that returns a *cobra.Command:

```go
func NewRootCmd() {
    return &cobra.Command {
        ...
    }
```

Reply:
Actually: what we can do here is move this root.go into it's own namespace, e.g. `package root` so the cmd/rukpakctl/main.go can call it via `root.NewCmd()`.